### ISSUE TITLE
[v3] Implement new Concept Exercise: basics

### ISSUE THREAD

**Issue by dector** [exercism/kotlin#435]
This issue describes how to implement the `basics` concept exercise for the Kotlin track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This exercise should teach bare minimum to help students understand Kotlin programs.

## Learning objectives

- Know what a variable is.
- Know how to define a variable.
- Know the difference between immutable and mutable variable.
- Know how to update a variable.
- Know how to define a method.
- Know how to return a value from a method.
- Know how to call a method.
- Know about basic types: Int, Unit.
- Know how to define single and multiline comments.

## Out of scope

- More advanced data types: String, Float/Double/Number, Nothing etc.
- Calling function with named parameters.
- Inline functions.
- Classes/Objects.
- Packages

## Concepts

- `basics`: know what a variable is; know how to define a variable; know the difference between immutable and mutable variable; know how to update a variable; know how to define a method; know how to return a value from a method; know how to call a method; know that methods must be defined in classes; know about basic types: Int, Unit; know how to define single and multiline comments.

## Prerequisites

_none_

## Resources to refer to

[Kotlin docs](https://kotlinlang.org/docs/reference/basic-syntax.html)

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
